test/cloudant.test.js

```diff
@@ -260,17 +260,33 @@ describe('cloudant connector', function() {
             done();
           });
       });
-      it('returns result when sorting type provided', function(done) {
+      it('returns result when sorting type provided - missing first level' +
+        'property', function(done) {
+        // Similar test case exist in juggler, but since it takes time to
+        // recover them, I temporarily add it here
```

Comment:
all test cases imported from juggler are skipped now, and I am recovering them: https://github.com/strongloop/loopback-connector-cloudant/issues/33